plugin/src/main/java/com/playmonumenta/scriptedquests/utils/MessagingUtils.java

```diff
@@ -10,8 +10,15 @@
 
 import com.playmonumenta.scriptedquests.Plugin;
 
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.TextComponent;
+import net.kyori.adventure.text.event.ClickEvent;
+import net.kyori.adventure.text.event.HoverEvent;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
+import net.kyori.adventure.text.serializer.plain.PlainComponentSerializer;
 import net.md_5.bungee.api.ChatColor;
-import net.md_5.bungee.api.ChatMessageType;
 /*
```

Comment:
This command about ChatMessageType is no longer needed, nor is the Spigot chat component API, if we're fully switched to Adventure. ChatColor isn't needed either for Adventure.